src/media-event-filter.ts

```diff
@@ -449,7 +449,14 @@ export const getMediaEventFilter = ({
   const onTimeupdate = (): void => {
     if (isNotReady()) return;
 
-    if (state.buffering && !mediaElement.paused) {
+    if (
+      state.buffering &&
+      !mediaElement.paused &&
+      // Playhead changed while not paused and buffering is ongoing
+      // which could indicate resumed playback. Check readyState
+      // to confirm
+      mediaElement.readyState === 4
```

Comment:
IMO it would be nice to use the constant instead `HTMLMediaElement.HAVE_ENOUGH_DATA`